friends/src/comp/FriendsList/FriendForm.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+class FriendForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newFriend: {
+        name: '',
+        age: '',
+        email: ''
+      }
+    };
+  }
+
+  handleChange = e => {
+    this.setState({
```

Comment:
Ideally you would pass in a function with previousState and reference that instead of referencing state directly.